src/Components/Navigation.js

```diff
@@ -21,30 +17,37 @@ const Navigation = ({
   const idRight = dataRight === undefined ? 1 : dataRight.number;
 
   return (
-    <StyledDV>
-      <StyledButtons type="button" onClick={() => onNavigate(idLeft)}>
+    <StyledNavigation>
+      <StyledNavigationButtons type="button" onClick={() => onNavigate(idLeft)}>
```

Comment:
This is only one button, so the name should reflect this, too.
```suggestion
      <StyledNavigationButton type="button" onClick={() => onNavigate(idLeft)}>
```